packages/core/src/lib/template/template.interface.ts

```diff
@@ -6,11 +6,16 @@ export interface ITemplate {
   messages: IMessage[];
 }
 
+export interface IMessageValidator {
+  validate(payload: ITriggerPayload): Promise<boolean> | boolean;
```

Comment:
@scopsy do you think we should provide template id as second param to be able to build more complex validators?

Reply:
I think we can start with only the payload for now